### DISCUSSION TITLE
[Feature]: Change Location of App Installs

### DISCUSSION THREAD

**Discussion by MoPaMo** [PlayCover/PlayCover#672] [Q&A] [answered]
### Is your feature request related to a problem?

Genshin is taking up to much space on my Macs intern SSD

### Describe the solution you'd like

I would love to be able to store apps in different locations, e. g. my external SSD

### Anything else?

I think this would improve overall quality by a lot

### Issue Language

- [X] Yes my issue is written in English

**Comment by TheMoonThatRises** [comment, marked as answer]
The app itself probably could be manually moved to an external SSD, but the main bulk of Genshin's data is in it's game files, which cannot be moved. Discussed in this discussion: https://github.com/PlayCover/PlayCover/discussions/473.

**Reply by mkfla1** [reply]
actually you can ln -s .../Containers/com.miHoYo.genshin/Data to your external SSD,
however you will lose your preference every time and the app will verify resource everytime

**Comment by ToustCZ** [comment]
Please, it's been a year already, add this feature. Lots of people use playcover to play genshin, because it's the only way on arm rn. We need this.

**Comment by Congyuwang** [comment]
You can do the following: 1. install macOS on your external drive. 2. install Genshin Impact on that external macOS. Works perfectly.

**Reply by macintoshyang** [reply]
seriously...but this might be a good idea if your native storage is too small.

**Comment by ghost** [comment]
There may be a solution, to use `ln -sF ~/Library/Containers/com.miHoYo.GenshinImpact /Volumes/your-disk` to link game data to your-disk external drive.

**Reply by iamindru** [reply]
does it work ?

**Comment by wps132230** [comment]
Another approach is provided in this discussion: https://github.com/PlayCover/PlayCover/discussions/1712
Basically, you could create a new volume in your external drive, move everything from the data folder of your Playcover game to the new volume, and then re-mount the volume back to the data folder.
